charts/fybrik/values.yaml

```diff
@@ -79,8 +79,8 @@ coordinator:
 
   # Configure the vault instance to be used by the coordinator manager
   vault:
-    # WARNING: it's an advance feature, set it to "false", if you all your modules and connectors do not require getting
-    # credentials to from Fybrik.
+    # WARNING: it's an advanced feature, set it to "false" if all your modules and connectors do not require getting
```

Comment:
Actually, I think the proper grammar is "if none of your modules and connectors requires..."

Reply:
as I know, both are correct